SokolTextGame/Commands.cs

```diff
@@ -50,7 +69,7 @@ public void Execute(string[] words, World world)
             else if (string.Join(" ", words) == "where am i") Console.Write(world.CurrentLocation().Description);
             else if (string.Join(" ", words) == "where can i go")
             {
-                string output = string.Join("", world.CurrentLocation().possibleLocation);
+                string output = string.Join(" ", world.CurrentLocation().possibleLocation);
```

Comment:
Change this to `string.Join(", ")` using a comma and a space, or otherwise it will look extremely weird and confusing to players if you just spit out several location names separated by just space. It doesn't look like human language this way.